src/app/data-planes/components/DataPlaneEntitySummary.vue

```diff
@@ -126,13 +126,12 @@
           </div>
         </section>
 
-        <section class="config-section">
-          <YamlView
-            id="code-block-data-plane-summary"
-            :content="dataPlane"
-            code-max-height="250px"
-          />
-        </section>
```

Comment:
Would it be possible for the moment to re-use this inside of ResourceCodeBlock? That way we keep a presentational component that just requires data to be passed into it.

Reply:
Not sure I understand your question. Re-use what inside `ResourceCodeBlock`? `YamlView`?

Reply:
Yeah, maybe I misunderstood the change set, but was YamlView a presentational component that didn't deal with any loading of things? If we want to load things in ResourceCodeBlock we could do that and then pass the result to YamlView, then we still have a presentational component to use.

This isn't a biggie though, especially if I misunderstood

Reply:
With what we settled on implementing, there isn’t a need for having a component that can handle both views. We might get back to this in a different form (i.e. without tabs) at some point.

And with that, it’s also true that YamlView was "more" presentational. In a sense ResourceCodeBlock is a more opinionated/application-specific component (hence the name implying its a code block for showing resources) so personally I’m fine with it being provided with a bit of logic to fetch the content it displays.